dockerfiles/base-ci-linux/Dockerfile

```diff
@@ -22,6 +22,9 @@ WORKDIR /builds
 
 # install tools and dependencies
 COPY clang9.key /etc/apt/trusted.gpg.d/debian-archive-jessie-automatic.gpg
+#download and install cmake
+ADD "https://github.com/Kitware/CMake/releases/download/v3.16.0/cmake-3.16.0-Linux-x86_64.sh" cmake.sh
```

Comment:
why not use https://packages.debian.org/buster/cmake ? it's version 3.13, is 3.19 really necessary, I think it will be easy to forget to update patch versions like that.

Reply:
Our Base Image is built on the stretch distribution. And there the maximum version is 3.9. C ++ 17 support has been included since version 3.10. Well, in this case, we take the version from the upstream, so it seems to me that this option is preferable

Reply:
@General-Beck I agree with @gabreal here, especially if a backport is available in Debian.

Reply:
Now, as we are on `buster`, we shouldn't install `cmake`, we have the right version built-in, right?

Reply:
@TriplEight you are correct.

Reply:
I thought we would use the upstream version of cmake. In CI then we will have a difference in versions on xenial, windows and osx.
xenial - 3.5.1
buster - 3.13.4
windows - 3.16
osx - 3.15
I think this option is the best in our case.

Reply:
I'd say `keep it simple`. Use the distro packages everywhere from the package managers we use (`apt`, `brew`, `chocolate`). It should be possible to agree on a minimum version and I assume versions should be downwards compatible. It's also giving me more confidence than running arbitrary shell scripts for creating our build environment. How can it be guaranteed that it will work well together with possible future updates of other packages? Who will be notified if there is a new upstream version with potential bug fixes?